Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:16.04
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends apt-utils git \
+                            libatlas-base-dev python-protobuf python-numpy \
+                            python-scipy python-h5py unzip make libblas-dev \
+                            liblapack-dev libatlas-base-dev gfortran \
+                            python-pip python-dev libprotobuf-dev \
+                            libleveldb-dev libsnappy-dev libopencv-dev \
+                            libhdf5-serial-dev libgflags-dev nodejs npm \
+                            libgoogle-glog-dev liblmdb-dev protobuf-compiler \
+                            cmake libboost-all-dev wget python-setuptools g++ \
+                            && rm -rf /var/lib/apt/lists/*
+
+ADD requirements/dev.txt requirements/common.txt /tmp/
+
+RUN pip install setuptools wheel \
```

Comment:
Can we not have all the pip requirements in the `requirements/dev.txt` or `requirements/common.txt`?

Reply:
How do You want to keep consistency both in Dockerfile and `requirements/dev.txt` etc? Just to get rid of the issues when someone would like to use this stuff without docker.